docs/_summary.md

```diff
@@ -100,6 +100,7 @@
       * [RGB Lighting](feature_rgblight.md)
       * [RGB Matrix](feature_rgb_matrix.md)
     * [Audio](feature_audio.md)
+    * [MIDI](feature_midi.md)
```

Comment:
Should be in alphabetical order.

Reply:
Figured it was logical to have MIDI after Audio, but I'll alphabetize it for consistency.